data/campaigns/Heir_To_The_Throne/achievements.cfg

```diff
@@ -77,6 +77,100 @@
+        hidden=yes
+        hidden_name= _ "???"
+        hidden_hint= _ "Violence is not the answer"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S11_Strike_First_Strike_Hard"
+        name= _ "Strike First Strike Hard"
+        name_completed= _ "Struck First Struck Hard"
+        description= _ "Defeat the orcish commander at the northern banks of Abez"
+        description_completed= _ "Defeated the orcish commander at the northern banks of Abez"
+        icon="units/orcs/warlord-defend-2.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/orcs/warlord-defend-2.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S11_Mirrored_Again"
+        name= _ "Mirrored Again"
+        name_completed= _ "Mirrored Again"
+        description= _ "Confront the scion of the Dark Queen once more"
+        description_completed= _ "Bested the scion of the Dark Queen yet again"
+        icon="data/campaigns/Heir_To_The_Throne/images/units/human-princess.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="data/campaigns/Heir_To_The_Throne/images/units/human-princess.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+        hidden_name= _ "???"
+        hidden_hint= _ "Do the unthinkable at <i>Ford of Abez</i>"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S11_Terror_to_the_Seas"
+        name= _ "Terror to the Seas"
+        name_completed= _ "Terror to the Seas"
+        description= _ "Defeat all spawned sea monsters in <i>Ford of Abez</i>"
+        description_completed= _ "Defeated all spawned sea monsters in <i>Ford of Abez</i>"
+        icon="units/monsters/kraken/kraken-defend1.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/monsters/kraken/kraken-defend2.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S13_Northern_Rebirth"
+        name= _ "Northern Rebirth"
+        name_completed= _ "Northern Rebirth"
+        description= _ "Defeat all orc commanders in <i>Dwarven Doors</i>"
+        description_completed= _ "Defeat all orc commanders in <i>Dwarven Doors</i>"
+        icon="data/campaigns/Heir_To_The_Throne/images/units/konrad-commander.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="data/campaigns/Heir_To_The_Throne/images/units/konrad-commander.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S13_From_the_Depths"
+        name= _ "From the Depths"
+        name_completed= _ "From the Depths"
+        description= _ "Defeat the Cuttlefish"
+        description_completed= _ "Defeated the cuttlefish"
```

Comment:
EIther capitalize or don't. Don't do both. (I'd recommend don't, personally.)

Reply:
made it Cuttlefish to both